chat-plugins/info.js

```diff
@@ -9,6 +9,8 @@
  *
  * @license MIT license
  */
+ 
+var request = require('request');
```

Comment:
By which is meant: you can do the same with just the ``http`` core module, @Irraquated 
See for reference: https://github.com/Zarel/Pokemon-Showdown/blob/master/chat-plugins/traditionalgames.js